walletnode/cmd/app.go

```diff
@@ -95,6 +98,10 @@ func NewApp() *cli.App {
 			return errors.Errorf("could not create temp-dir %q, %w", config.TempDir, err)
 		}
 
+		if err := os.MkdirAll(config.RqFilesDir, os.ModePerm); err != nil {
```

Comment:
could you put directly `0777` than os.ModePerm? Because os.ModePerm is usually used as bit mask than directly.